<launch> 
  <param name="robot_description"
    command="$(find xacro)/xacro $(find mit_cheetah_description)/urdf/cheetah.urdf.xacro"/>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>

  <node pkg="quadruped_controller" type="gait_visualizer" name="gait_visualizer" output="screen">
    <rosparam command="load" file="$(find quadruped_controller)/config/gait_visualizer.yaml" />
  </node>

  <node name="rviz" pkg="rviz" type="rviz" required="true" args="-d $(find quadruped_controller)/config/gait_visualizer.rviz"/>  
</launch>
